appIos/appIos.xcodeproj/project.pbxproj

```diff
@@ -344,6 +344,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 5G893F5HW7;
 				ENABLE_PREVIEWS = YES;
+				"EXCLUDED_ARCHS[sdk=*]" = "";
```

Comment:
I'm not sure why has this been added, can remove it

Reply:
Not sure what it means, will check…

Reply:
I don't think this should be here, interesting why Xcode added it - lets revert it from git

Reply:
Removed